roles/repo_setup/tasks/sync_repos.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Copy generated repos to /etc/yum.repos.d directory
+  when: not cifmw_repo_setup_output.startswith(ansible_user_dir)
+  block:
+    - name: Find existing repos from /etc/yum.repos.d directory
+      ansible.builtin.find:
+        paths: "/etc/yum.repos.d/"
+        patterns: "*.repo"
+        recurse: false
+      register: _yum_repos
+
+    - name: Remove existing repos from /etc/yum.repos.d directory
+      become: "{{ not cifmw_repo_setup_output.startswith(ansible_user_dir) }}"
```

Comment:
hmmm since you're calling `sudo` you want to remove it and put `become: true` I think?

Same goes for the 2 other occurrences bellow.